src/style.css

```diff
@@ -0,0 +1,46 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars__block {
+  display: flex;
+}
+
+.star {
+  width: 15px;
+  height: 15px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__block.stars--5 .star {
```

Comment:
- one BEM block shouldn't style another BEM block
- do you really need stars__block here?